### DISCUSSION TITLE
Challenge Arena

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#93] [General]
I thought it could be cool to have a Single Player Challenge Arena.

Where you enter the arena and every 10min or whatever, you have to do a random challenge. And then you get or lose points depending on your progress in the challenge

This could be interesting because I have a snake that can solve a few challenges but only every so often when it gets lucky. Right now I’ve gotten the medal for those challenges because I’ve tried them a bunch. But it could be interesting to have a reason to improve how my snake can handle the challenges and make it work a higher percentage of the time

**Comment by InterestingBrainPoops** [comment]
I love the idea since it encourages people to have snakes that perform well consistently, similar to the global arenas. I can see this going a long way to improving the quality of the challenge snakes!

**Comment by bvanvugt** [comment]
Haha! Sort of like a challenge gauntlet arena. I love this!
